tests/test_api_system.py

```diff
@@ -55,7 +54,7 @@ def test_02_update_config(self):
                                            headers={'Authorization': self.at}):
             res = self.app.full_dispatch_request()
             self.assertTrue(res.status_code == 200, res)
-            self.assertTrue('"key3": "update"' in res.data, res.data)
+            self.assertIn('key3', res.json['result']['value'], res.json)
```

Comment:
Hm, I think this test doesn't check if ``res.json['result']['value']['key3'] == "update"``, like the old test, but maybe that's intentional?

Reply:
True, got mixed up in the json format. Will fix.

Reply:
fixed in 21bdef5b